webapp/channels/src/components/widgets/icons/fa_next_icon.tsx

```diff
@@ -8,21 +8,17 @@ import LocalizedIcon from 'components/localized_icon';
 import {t} from 'utils/i18n';
 
 type Props = {
-    additionalClassName: string | null;
+    additionalClassName?: string;
 }
 
-export default class NextIcon extends React.PureComponent<Props> {
-    public static defaultProps: Props = {
-        additionalClassName: null,
-    };
-
-    public render(): JSX.Element {
-        const className = 'icon icon-chevron-right' + (this.props.additionalClassName ? ' ' + this.props.additionalClassName : '');
-        return (
-            <LocalizedIcon
-                className={className}
-                title={{id: t('generic_icons.next'), defaultMessage: 'Next Icon'}}
-            />
-        );
-    }
+function NextIcon(props: Props) {
```

Comment:
Again, the same changes as in https://github.com/mattermost/mattermost/pull/24787